src/components/common/TmBalance.vue

```diff
@@ -19,6 +19,24 @@
                 {{ overview.totalStake | bigFigureOrShortDecimals | noBlanks }}
               </h2>
             </div>
+            <div class="currency-selector">
+              <img
+                class="currency-flag"
+                :src="
+                  '/img/icons/currencies/' +
+                    selectedFiatCurrency.toLowerCase() +
+                    '.png'
+                "
+                alt="USA flag"
```

Comment:
the alt tag should change with the `selectedFiatCurrency`

Reply:
oh true, I forgot!